.github/workflows/release.yml

```diff
@@ -0,0 +1,90 @@
+name: ReleaseCmdStanPy
+
+on:
+  workflow_dispatch:
```

Comment:
Ok, this is very cool feature (also reason for my confusion, first time I hear you can add input manually)

Reply:
I'm also quite new to Github Actions, it's quite nice.